projects/plugins/jetpack/_inc/lib/core-api/wpcom-endpoints/class-wpcom-rest-api-v2-endpoint-app-media.php

```diff
@@ -0,0 +1,184 @@
+<?php
+/**
+ * REST API endpoint for the media uploaded by the Jetpack app.
+ *
+ * @package automattic/jetpack
+ * @since 13.0.0
```

Comment:
```suggestion
 * @since $$next-version$$
```

This will be automatically replaced by the correct version on release.